### PR TITLE
polymer++

### PR DESCRIPTION
**Pull request by notwaldorf**
`x-style` -> `custom-style`
`mixin` -> `@apply`
`version++`
`polymer++`

### CODE REVIEW

File: paper-icon-button.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
 <link rel="import" href="../paper-behaviors/paper-button-behavior.html">
 <link rel="import" href="../paper-ripple/paper-ripple.html">
 
-<style is="x-style">
+<style is="custom-style">
   * {
     --paper-icon-button-disabled-text: var(--disabled-text-color);
```
Comment (cdata) [inline paper-icon-button.html]: This should be changed to a default value below.

**Comment by notwaldorf** [comment]
Punting on the inline defaults for now.

**Comment by cdata** [comment]
Oh no you don't!

**Comment by cdata** [comment]
LGTM :+1:

**Pull request merged**